site/docs/guide/context.md

```diff
@@ -400,12 +400,17 @@ If you have different [additive context flavors](#additive-context-flavors), you
 type MyContext = Context & FlavorA & FlavorB & FlavorC;
 ```
 
+The order of context flavors does not matter, you can combine them in any order you like.
+
 Multiple [transformative context flavors](#transformative-context-flavors) can also be combined:
 
 ```ts
 type MyContext = FlavorX<FlavorY<FlavorZ<Context>>>;
 ```
 
+Here, the order could matter, as `FlavorZ` transforms `Context` first, then `FlavorY`, and the result of this will be transformed again by `FlavorZ`.
```

Comment:
```suggestion
Here, the order could matter, as `FlavorZ` transforms `Context` first, then `FlavorY`, and the result of this will be transformed again by `FlavorX`.
```

The final flavor should be `FlavorX`

Reply:
Thanks, you're right